### ISSUE TITLE
Allow to use GitHub Integration without a GitHub App

### ISSUE THREAD

**Issue by pascal-hofmann** [cloudflare/pint#490]
Prior to v0.39 it was possible to use the GitHub integration without a GitHub App. This allowed us to run pint in jenkins and report errors back to pull requests in our GitHub Enterprise instance. With #486 being merged this is not possible anymore:

```
23:08:48  level=fatal msg="Execution completed with error(s)" error="submitting reports: failed to create a new check run: POST https://github.example.com/api/v3/repos/our-org/our-repo/check-runs: 403 You must authenticate via a GitHub App. []"

```

It would be great to have this back again.

PS: Thanks for this great tool! 🤩👍

**Comment by LukoJy3D** [comment]
We are running into the same issue after v0.39. However, the author confirmed that it should work without any app: https://github.com/prymitive/pint-action/issues/26#issuecomment-1363990642

**Comment by prymitive** [comment]
@pascal-hofmann  can you share your workflow file please?

**Comment by prymitive** [comment]
Also: is your PR raised from the same repository or from a fork?

**Comment by prymitive** [comment]
Just released [v0.40.0](https://github.com/cloudflare/pint/releases/tag/v0.40.0) that reverts back to old GitHub API. 
But added some changes to make avoid issues like #478.

**Comment by pascal-hofmann** [comment]
Hi. Thanks for the revert and sorry for the late feedback. It works like a charm again.

We run pint via Jenkins, so there is no workflow file.
